libs/coin-modules/coin-evm/src/__tests__/coin-tester/scenarios/ethereum.ts

```diff
@@ -0,0 +1,252 @@
+import Eth from "@ledgerhq/hw-app-eth";
+import { BigNumber } from "bignumber.js";
+import { ethers, providers } from "ethers";
+import { killSpeculos, spawnSpeculos } from "@ledgerhq/coin-tester/signers/speculos";
+import { encodeTokenAccountId } from "@ledgerhq/coin-framework/account/index";
+import { Scenario, ScenarioTransaction } from "@ledgerhq/coin-tester/main";
+import { getCryptoCurrencyById } from "@ledgerhq/cryptoassets/currencies";
+import { buildAccountBridge, buildCurrencyBridge } from "../../../bridge/js";
+import { makeAccount } from "../../fixtures/common.fixtures";
+import { EvmNftTransaction, Transaction as EvmTransaction } from "../../../types";
+import resolver from "../../../hw-getAddress";
+import { setCoinConfig } from "../../../config";
+import {
+  ethereum,
+  ERC20Interface,
+  USDC_ON_ETHEREUM,
+  ERC721Interface,
+  ERC1155Interface,
+} from "../helpers";
+import { clearExplorerAppendix, getLogs, setBlock } from "../indexer";
+import { killAnvil, spawnAnvil } from "../anvil";
+
+const scenarioSendTransaction: ScenarioTransaction<EvmTransaction> = {
+  name: "Send ethereum",
+  amount: new BigNumber(100),
+  recipient: "0x6bfD74C0996F269Bcece59191EFf667b3dFD73b9",
+  expect: account => {
+    const { operations } = account;
+    const sendOperation = operations.find(operation => operation.transactionSequenceNumber === 0);
+    expect(sendOperation?.transactionRaw?.amount === "100").toBeDefined();
+  },
```

Comment:
Not sure this test is really what we want to verify. I think we can check 2 things:
- Account should have 1 more operation compared to our initial situation (I guess account has 3 operations at the beginning, so account.operations.length should now be 4 ?).
- Latest operation has type OUT, value is 100 + operation.fee
- Balance is now 10000^18 (because we took an account initialized by anvil with 10k ETH balance) - 100 (operation value) - operation.fee (will be a little bit different when we're testing L2s, but that's a problem for tomorrow :p)